packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -239,28 +280,21 @@ export async function deleteDatasetPrompt(datasetProvider: IZoweTree<IZoweDatase
 
     // Confirm that the user really wants to delete
     globals.LOG.debug(localize("deleteDatasetPrompt.log.debug", "Deleting data set(s): ") + deletedNodes.join(","));
-    const quickPickOptions: vscode.QuickPickOptions = {
-        placeHolder: localize(
-            "deleteDatasetPrompt.quickPickOption",
-            "Delete data set(s)? This will permanently remove them from your system."
-        ),
-        ignoreFocusOut: true,
-        canPickMany: false,
-    };
-    if (
-        (await vscode.window.showQuickPick(
-            [
-                localize("deleteDatasetPrompt.showQuickPick.delete", "Delete"),
-                localize("deleteDatasetPrompt.showQuickPick.Cancel", "Cancel"),
-            ],
-            quickPickOptions
-        )) !== localize("deleteDatasetPrompt.showQuickPick.delete", "Delete")
-    ) {
-        globals.LOG.debug(
-            localize("deleteDatasetPrompt.showQuickPick.log.debug", "Dataset deletion action was canceled.")
-        );
-        return;
-    }
+    const deleteButton = localize("deleteDatasetPrompt.confirmation.delete", "Delete");
+    const message = localize(
+        "deleteDatasetPrompt.confirmation.message",
+        "Are you sure you want to delete these items?\nThis will permanently remove these data sets and/or members from your system.\n\n{0}",
```

Comment:
@JillieBeanSim What would you think about adding an item count in this message like how VS Code does theirs? E.g. "Are you sure you want to delete these 30 items?"
Adding the count of items to be deleted can help the user quickly assess if they have accidentally selected more (or less) than the expected number of items for deletion. 
I think it shouldn't be too hard to add; you might be able to get the count just by doing `deletedNodes.length`:

```
    const message = localize(
        "deleteDatasetPrompt.confirmation.message",
        "Are you sure you want to delete these {0} items?\nThis will permanently remove these data sets and/or members from your system.\n\n{1}",
        deletedNodes.length,
        deletedNodes.toString().replace(/(,)/g, "\n")
    );
```

In fact, VS Code is a bit fancier with generating different messages based on if you have a file(s), folder(s), or mix selected, and if you have one versus multiple selected. So we can consider additional refinements like those in future PRs.

Reply:
@lauren-li `these 1 items` will look funny, maybe it should be `the following items`?

Reply:
@JillieBeanSim Mostly I wanted to have the item count in the message. We could write: `Are you sure you want to delete the following {0} item(s)?"

Reply:
@lauren-li it's in now